### ISSUE TITLE
Feature Request:  Allow change of LED color via application or in configuration file?

### ISSUE THREAD

**Issue by Nodi-Rubrum** [SIRprise/SSD-LED#9]
Feature Request:  Allow change of LED color via application or in configuration file?

**Comment by SIRprise** [comment]
Good idea! I will think about it, how to realize that. A problem could be mix-colors with additive color mixing if you use two different, dark color it could result in a more bright color if I think right.

**Comment by Nodi-Rubrum** [comment]
Maybe use a 'gradient' scale or range?  Or complimentary colors?  Just ideas.  I would love light blue for reads and dark blue for writes for example, just my idea.

**Comment by SIRprise** [comment]
I like that idea and I like blue! :-)
That would also result in a less distractive icon I think.
At the moment I am a bit busy, but I will have time at the end of the month and in november and will add that feature!

**Comment by Nodi-Rubrum** [comment]
Sounds good!

**Comment by SIRprise** [comment]
I just released a new version with the color settings feature + save/load color config

**Comment by SIRprise** [comment]
I reopened the issue, because I hope anyone of you have tips how to do the color mixing.
At the moment I sum up the RGB-values of the read and write color - and the values there are maximum values. In a first approach I used the average of both, but that means a color reaches only half of the selected brightness, so I didn't use that approach.
There could also be a problem if the background/default-color is not set to black. The background color doesn't matter in the calculation for now.
One solution could be to disable the mixing (via checkbox in the settings) and use the threshold of the KBytes/s selector to configure.
